src/features/wallet/send/SendView.tsx

```diff
@@ -194,11 +193,16 @@ const SendView = ({
       let amount = ''
       let balanceAmount = new Balance(0, CurrencyType.networkToken)
       if (scanAmount) {
+        const floatAmount = parseFloat(scanAmount.split(',').join(''))
```

Comment:
is this just attempting to remove any comma formatting? you could do `scanAmount.replace(/,/g, '')` rather than splitting and rejoining

Reply:
Yeah, sounds good let me update it

Reply:
This is for the QR data, which we will require in US format so we can manually parse out the `,` which parseFloat doesnt work with